### DISCUSSION TITLE
Des saisons pour les guildes?

### DISCUSSION THREAD

**Discussion by aureochocob0n** [Crownicles/Crownicles#2575] [Nouvelles idées]
Je propose de, tout comme les saisons de combats, réaliser des saisons pour les guildes. 
Ainsi, nous pourrions aussi avoir des récompenses pour un bon classement hebdomadaire ou mensuel.
Cela permettrait d'apporter aux guildes une nouvelle compétitivité et d'encourager les joueurs à faire l'île

**Comment by Cirpacha** [comment]
oh pas mal
si y a ça jvoudrais bien pareil pour les points, des récompenses qui augmentent avec le nb de points qu'on fait en une saison, ils servent un peu à rien sinon
